pkg/component/operator/web/v0/crawl_website.go

```diff
@@ -38,9 +41,12 @@ type CrawlWebsiteInput struct {
 	MaxDepth int `json:"max-depth"`
 }
 
-func (i *CrawlWebsiteInput) Preset() {
-	if i.MaxK < 0 {
-		i.MaxK = 0
+func (i *CrawlWebsiteInput) preset() {
+	if i.MaxK <= 0 {
+		// When the users set to 0, it means infinite.
+		// However, there is performance issue when we set it to infinite.
+		// So, we set the default value to solve performance issue easily.
+		i.MaxK = 8000
```

Comment:
This limitation should be documented in the public readme.

Reply:
Do you mean to make it in `README.mdx`?

This setting is just to fix the performance issue. So, the users don't have to know this information. Mainly, it is for developer. I will add clearer comments here.

Reply:
I updated [here](https://github.com/instill-ai/pipeline-backend/pull/772/commits/c120df686c18a8618f0d565be751f0c2c7e89729).